events/show-and-tell.md

```diff
@@ -5,27 +5,24 @@ description: Internal presentations with no preparation
 
 # Show & Tell
 
-Show & Tell is a new, **optional** weekly event we're trying out to get more visibility into what our teammates
-have been working on recently. It is scheduled weekly for a half hour each week (see the shared Engineering Open
-Meetings calendar) and consists of three ten-minute walkthroughs. These walkthroughs are a presentation of
-works-in-progress and _no_ preparation is expected of these presenters. We get people excited about Show & Tell at
-[Monday standups](open_standup.md) and then do an _@here_ an hour beforehand to remind people. We then meet and, in
-an open mic-style, developers step forward to show and tell!
+In order to better know what our teammates are up to, every other week we set
+aside an hour and in an open mic-style setting show each other what we're
+working on. These short talks/demos/code reviews are not meant to be polished
+presentations, but instead impromptu ways we can share lessons learned and cool
+things discovered!
+
+This happens at the same time slot as [Lunch & Learn](lunch-and-learn.md) and we
+alternate between the two events. On Lunch & Learn weeks without a speaker lined
+up, we fall back to doing a bonus Show & Tell.
```

Comment:
Nice!

This diff mostly looks like line-length noise, but the bits here and above LGTM 👍